pyvista/utilities/geometric_objects.py

```diff
@@ -432,6 +432,10 @@ def Line(pointa=(-0.5, 0.0, 0.0), pointb=(0.5, 0.0, 0.0), resolution=1):
     resolution : int, optional
         Number of pieces to divide line into.
 
+    points : np.ndarray or list, optional
+        List of points defining a broken line, default is ``None``
+        If given, pointa, pointb and resolution will be ignored.
```

Comment:
Then I think this should be a separate function altogether. I do not like having one new argument that renders all other existing arguments useless